### ISSUE TITLE
Would you please release the coco pre-trained models? 

### ISSUE THREAD

**Issue by pppplin** [jiaxi-wu/MPSR#15]
Or did you run any experiments on COCO for less than 10 shots? Also curious on how the 10/30 shots number changes exactly after replacing validation set, would you please report the changed numbers? Thanks in advance!

**Comment by jiaxi-wu** [comment]
1. I will update a pre-trained model on COCO later.
2. Sorry, I haven't run experiments on COCO for less than 10 shots. You can test them by yourself.
3. The changed numbers are output when you run `tools/fewshot_exp/datasets/coco_create_standard.py`:
```
# image number afer removing invalid images, 10shot
384 
# instance numbers after removing invalid images, 10shot
[10, 10, 10, 10, 10, 6, 8, 10, 10, 10, 9, 10, 10, 10, 9, 10, 8, 10, 10, 9, 10, 8, 10, 9, 10, 10, 10, 9, 10, 10, 10, 10, 10, 10, 10, 8, 9, 10, 10, 10, 10, 10, 9, 10, 10, 10, 10, 10, 10, 10, 9, 10, 10, 10, 10, 10, 8, 10, 10, 9, 10, 10, 9, 8, 9, 10, 8, 9, 9, 8, 10, 10, 9, 10, 10, 9, 10, 10, 10, 10]
# image number after adding random samples, 10shot
412
# instance numbers after adding random samples, 10shot
[10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10]
# image number afer removing invalid images, 30shot
1146
# instance numbers after removing invalid images, 30shot
[29, 30, 28, 29, 30, 30, 30, 30, 30, 29, 27, 30, 29, 29, 30, 30, 29, 30, 24, 30, 29, 30, 29, 29, 30, 28, 29, 29, 28, 26, 29, 27, 27, 28, 27, 27, 26, 29, 29, 28, 30, 30, 28, 30, 28, 30, 30, 30, 29, 30, 30, 30, 29, 28, 30, 29, 28, 28, 29, 30, 27, 30, 29, 28, 27, 30, 29, 30, 30, 30, 29, 30, 27, 30, 30, 30, 29, 30, 30, 30]
# image number after adding random samples, 30shot
1199
# instance numbers after adding random samples, 30shot
[30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 29, 30, 30, 29, 30, 29, 30, 30, 27, 27, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 29, 30, 30, 30, 30, 30, 30, 30, 30, 30]
```

**Comment by pppplin** [comment]
> 29

many thanks! I know this might be too much to ask but will you upload the COCO pre-trained model within few days? I'm working on an approaching deadline and if you can't make it within few days I'll run it by myself. Thanks again!!

**Comment by jiaxi-wu** [comment]
As the performance is stable, I recommend you to run it first.

**Comment by hanhao2019102935** [comment]
Thanks to the author's great work, I am also waiting for the pre-training model of Coco
